packages/libs/error-stack/src/lib.rs

```diff
@@ -299,12 +299,7 @@
 //! return an iterator of all provided values with the specified type. The value, which was provided
 //! most recently will be returned first.
 //!
-//! **Currently, the API has not yet landed in `core::any`, thus in the meantime it has been
-//! included in the library implementation and is available at [`error_stack::provider`]. Using it
-//! requires a nightly compiler.**
-//!
 //! [`attach`]: Report::attach
-//! [`error_stack::provider`]: crate::provider
 //! [`Provider` API]: https://rust-lang.github.io/rfcs/3192-dyno.html
```

Comment:
Is this still needed?

Reply:
This was only used in the above comment. If it would have been needed, the docs-test would fail.

Reply:
I'm on about the the [`Provider` API] link, not the removed line for `error_stack::provider`. I don't think docs-tests would fail if you add an unused link?